core/logger/prettyconsole.go

```diff
@@ -50,7 +50,7 @@ func (pc PrettyConsole) Write(b []byte) (int, error) {
 // Close is overridden to prevent accidental closure of stderr/stdout
 func (pc PrettyConsole) Close() error {
 	switch pc.Sink {
-	case os.Stderr, os.Stdout:
+	case os.Stderr, os.Stdout, stderrWriter{}:
```

Comment:
Should be able to drop this if Close() is a no-op anyways:
```suggestion
	case os.Stderr, os.Stdout:
```